lib/error/error_manager_impl.cpp

```diff
@@ -34,10 +34,12 @@ void ErrorManagerImpl::raise_error(const Error& error) {
     if (validate_error_types) {
         if (std::find(allowed_error_types.begin(), allowed_error_types.end(), error.type) ==
             allowed_error_types.end()) {
-            throw EverestArgumentError("Error type " + error.type + " is not allowed to be raised.");
+            EVLOG_error << "Error type " << error.type << " is not allowed to be raised. Ignoring.";
+            return;
         }
         if (!this->error_type_map->has(error.type)) {
-            throw EverestArgumentError("Error type " + error.type + " is not known.");
+            EVLOG_error << "Error type " << error.type << " is not known.";
+            return;
```

Comment:
This if statement doesn't really make sense at all, it should be removed, since an error_type which is in `allowed_error_types` should be in ` error_type_map`, too.

As a replacement for this check it would make sense to validate all types in ` allowed_error_types` once in constructor

Reply:
Sounds good, can you change this?